src/chrome/content/rules/ING.org.xml

```diff
@@ -0,0 +1,13 @@
+<!--
+	Invalid certificate:
+	- dipl.ing.org
```

Comment:
Here is a wildcard DNS record... Every subdomains that don't actually exist are redirected to https://ing.org/wp-signup.php?new=$DOMAIN